hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManagerServiceGrpc.java

```diff
@@ -126,20 +127,19 @@ public void submitRequest(OMRequest request,
       omResponse =
           UserGroupInformation.getCurrentUser().doAs(
```

Comment:
Should nt be this with actual user passed from protobuf?
And also even now rpc also used to pass request with s3 credentials still do we need it?

Reply:
@bharatviswa504, yes as discussed previously offline, the s3 creds processed by the` serverSideProtocol`  `submitRequest`.